test/e2e-go/restAPI/simulate/simulateRestAPI_test.go

```diff
@@ -338,15 +338,16 @@ int 1`
 		a.LessOrEqual(followerSyncRound+i+1, binary.BigEndian.Uint64((*result.TxnGroups[0].Txns[0].Txn.Logs)[0]))
 	}
 
-	// There should be a failure when the round is too far back
+	// If the round is too far back, we should get an error saying so.
 	simulateRequest.Round = basics.Round(followerSyncRound - 3)
-	require.Eventually(t, func() bool {
-		// use Eventually to workaround the ledger delays with committing accounts
-		result, err = simulateTransactions(simulateRequest)
-		return err != nil
-	}, 6*time.Second, 500*time.Millisecond)
-
-	a.Error(err)
+	result, err = simulateTransactions(simulateRequest)
```

Comment:
I guess it would be better to write a manual loop with few attempts and then bail if err is still empty (instead of erring as Eventually does)

Reply:
Good idea, I added that in 6734feaed